YapDatabase/YapDatabase.m

```diff
@@ -882,37 +882,42 @@ - (BOOL)configureEncryptionForDatabase:(sqlite3 *)sqlite
                 return NO;
             }
         }
-        
-        //Setting the PBKDF2 default iteration number (this will have effect next time database is opened)
-        if (options.cipherDefaultkdfIterNumber > 0) {
-            char *errorMsg;
-            NSString *pragmaCommand = [NSString stringWithFormat:@"PRAGMA cipher_default_kdf_iter = %lu", (unsigned long)options.cipherDefaultkdfIterNumber];
-            if (sqlite3_exec(sqlite, [pragmaCommand UTF8String], NULL, NULL, &errorMsg) != SQLITE_OK)
-            {
-                YDBLogError(@"failed to set database cipher_default_kdf_iter: %s", errorMsg);
-                return NO;
+
+        BOOL hasCustomCipherConfiguration = NO;
+        if (options.cipherDefaultkdfIterNumber > 0 || options.kdfIterNumber > 0 || options.cipherPageSize) {
+            hasCustomCipherConfiguration = YES;
```

Comment:
In this hunk, everything after is a whitespace change. We just want to track whether we've customized any cipher params. Signal doesn't do this, but other YapDB users do.